src/ResourceManager/Compute/ChangeLog.md

```diff
@@ -20,9 +20,9 @@
 ## Current Release
 
```

Comment:
@hyonholee can you make sure there is no space between the header and the first bullet?